app/views/monitoring.html

```diff
@@ -190,12 +190,12 @@ <h3>
                     <div class="list-pf-additional-content">
                       <div class="list-pf-additional-content-item">
                         <div class="pods">
-                          <a href=""
+                          <span
```

Comment:
This fixes the right-click behavior, but unfortunately it breaks keyboard and screenreader accessibility :/

We could leave it a link and then implement a filter that creates the URL.

```html
<a ng-href="replicationController | donutURL : podsByOwnerUID[replicationController.metadata.uid]" ...>
```

The filter would do what the current Navigate method does, except it just returns a URL without changing location. Then open in new window should work.

Reply:
How do we handle the case where we [navigate directly to the pod if there is only 1](https://github.com/rhamilto/origin-web-console/blob/e362dc49cbf613150b4ff1a84904bd2eb5cbdd59/app/scripts/services/navigate.js#L161-L163)?

Reply:
The filter can return a different URL if there's one pod.